regex-assembly/942190.ra

```diff
@@ -3,25 +3,27 @@
 
 ##!+ i
 
-[\"'`]\s*?!\s*?[\"'`\w]
-[\"'`];?\s*?having\b\s*?[^\s]
-[\"'`];?\s*?select\b\s*?[^\s]
-[\"'`];?\s*?union\b\s*?all\b\s*?[^\s]
-[\"'`];?\s*?union\b\s*?distinct\b\s*?[^\s]
-[\"'`];?\s*?union\b\s*?select\b\s*?[^\s]
+##!> define quotes [\"'`]
+
+{{quotes}}\s*!\s*[\"'`\w]
```

Comment:
Use an assemble block with `{{quotes}}` as prefix so you don't have to repeat it on each of these lines:
```
##!> assemble
  {{quotes}}
  ##!=>
  ...
##!<
```